docs/guide-about.md

```diff
@@ -4,62 +4,59 @@ title: About SerialPort
 slug: /
 ---
 
-## Quick Answers to Important Questions
-- **For support**, open a [GitHub issue](https://github.com/serialport/node-serialport/issues/new/choose).
+## Quick Answers
+
+- **For support**, you're in the right place. Have a look around and if you're stuck open a [GitHub issue or Discussion](https://github.com/serialport/node-serialport/issues/new/choose).
```

Comment:
we could add a separate link for discussions : https://github.com/serialport/node-serialport/discussions/new

however it might be worth clarifying when people should log an issue vs discussion.  I also wonder if we should simply link to the issues, rather than creating a new one as it might encourage people to check before logging a dupe

Reply:
ah, I dint realise that last item was a link to discussions... can we move it to the top of the list?